gpustack/worker/serve_manager.py

```diff
@@ -252,28 +511,27 @@ def _start_serve_process(self, mi: ModelInstance):  # noqa: C901
                         )
                         mi.ports.append(connecting_port)
 
-            logger.info(
+            logger.debug(
                 f"Starting model instance {mi.name}"
-                f"{'' if not is_main_worker else f'on port {mi.ports if mi.ports else [mi.port]}'}"
+                f"{'' if not is_main_worker else f' on ports {mi.ports if mi.ports else [mi.port]}'}"
             )
 
             process = multiprocessing.Process(
-                target=ServeManager.serve_model_instance,
+                target=ServeManager._serve_model_instance,
                 args=(
                     mi,
                     backend,
                     self._clientset.headers,
                     log_file_path,
                     self._config,
                     self._worker_id,
-                    self.inference_backend_manager.get_backend_by_name(backend),
+                    self._inference_backend_manager.get_backend_by_name(backend),
                 ),
             )
             process.daemon = False
             process.start()
```

Comment:
Keeping fork a process to provisioning a workload, considering there is still an I/O works inside.